packages/restlessness-cli/src/index.ts

```diff
@@ -7,6 +7,11 @@ import dev from './commands/dev';
 import createEnv from './commands/create-env';
 
 const cli = async () => {
+  const majorVersion: number = parseInt((/^(\d+)(\.\d+){0,2}$/.exec(process.versions.node))[1], 10);
+  if (majorVersion < 12) {
+    throw new Error('Run command requires node version >= 12.x');
```

Comment:
This should be changed to "Dev command" right?

Reply:
I moved the check on the root of the cli because I thought that the check should happens for every command, i.e. a user should be able to use all commands or none.
Do you think it is ok?

Reply:
Yes sorry I misread the file name. Yes I agree, so instead of "Run command" we can put a generic "Restlessness" maybe

Reply:
Right! I missed the message😆